musicname/ze_dark_souls_ptd_v0_4.cfg

```diff
@@ -0,0 +1,23 @@
+"music"
+{
+    "anor_londo_music_1_1.mp3"          "Dark Souls 3 OST - Twin Princes"
+  //"anor_londo_music_2_1.mp3"          ""
```

Comment:
According to legacy config, `anor_londo_music_2_1.mp3` should be `"Demon's Souls OST - Maiden Astraea (Hellkite Remix)"`. Will need to double check.

Reply:
looks right, didnt check old config